### PR TITLE
abracadabra: init at 2.1.1

### PR DESCRIPTION
**Pull request by markuskowa**
###### Description of changes
DAB/DAB+ radio application for RTL-SDR, Airspy, and SoapySDR supported radios with Qt6 GUI.

###### Things done
Tested with RTL-SDR and Airspy mini.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/radio/abracadabra/default.nix [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    qtmultimedia
+    qttools
```
Comment (sikmir) [inline pkgs/applications/radio/abracadabra/default.nix:26]: `qttools` to `nativeBuildInputs`?

**Comment by sikmir** [comment]
Result of `nixpkgs-review pr 234312` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>abracadabra</li>
  </ul>
</details>

File: pkgs/applications/radio/abracadabra/default.nix (hunk 1 above)
Comment (markuskowa) [reply pkgs/applications/radio/abracadabra/default.nix]: Yes, works.

**Comment by sikmir** [comment]
Is it working for you? I've tried with my rtl-sdr dongle and nothing scanned:(

**Comment by markuskowa** [comment]
> Is it working for you? I've tried with my rtl-sdr dongle and nothing scanned:(

Yes, I had no problem using. It works perfectly fine. The scan started automatically (takes a few minutes). You can also set the channel manually if you go to settings and switch the UI to "expert mode".
Did it not scan for you or not find any station?

**Review by sikmir — APPROVED** [review]

**Pull request merged**